test/functional/database-schema/sequences/entity/Person.ts

```diff
@@ -0,0 +1,16 @@
+import {Entity} from "../../../../../src/decorator/entity/Entity";
+import {PrimaryGeneratedColumn} from "../../../../../src/decorator/columns/PrimaryGeneratedColumn";
+import {Column} from "../../../../../src/decorator/columns/Column";
+
+@Entity()
+export class Person {
+
+    @PrimaryGeneratedColumn()
+    Id: number;
+
+    @Column({
+        unique: true
+    })
+    Name: string;
```

Comment:
can you please use camelCase in property names in this, others and future models, its its more natural to use camelCase in js world.

Reply:
Sure, I did it this way since this is how I discovered the issue. So instead of using different database names in the decorator, I chose to take the class as is in my code